core/src/modules/configLoaders/index.ts

```diff
@@ -129,8 +128,24 @@ export async function processConfigObjects(
       configObjects = sortConfigurationObjects(configObjects);
     }
   } catch (error) {
-    // the config loaders below will produce better error messages
-    sortError = error;
+    errors.push(error);
```

Comment:
I note below that `errors` has the `message` string and this is the whole object. should it be `error.message`?

Reply:
Yes! Found that when looking to clean up the specs.